irakli_iobashvili_week_2/Challenge #2 - Github Api/index.php

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://unpkg.com/@picocss/pico@latest/css/pico.min.css">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <main>
+    <h1>Enter Github username:</h1>
+    <div class="form-class">
+      <form action="" method="post">
+        <div class="belowPadding">
+        <input type="text" name="username">
+        <input type="checkbox" name="repos"  value="repositories">
+        <label for="repos">Repositories</label>
+        <input type="checkbox" name="followers"  value="followers">
+        <label for="repos">followers</label>
+      </div>
+      <button type="submit">Submit</button>
+    </form>
+  </div>
+    <?php 
+      include 'functions.php';
+      if(isset($_POST["username"])){
+        $username = $_POST['username'];
+        $pages = getPages($username);
+        $num = 1;
+      }
+      $headers = [
+        'User-Agent: GitHub-username'
+        ];
+        // Get how many public repositories user has
+          ?>
+        <?php if($username ?? null): ?>
+          <table>
+            <thead>
+              <tr>
+                <th>No.</th>
+                <th>ID</th>
+                <th>Name</th>
+                <th>Description</th>
+              </tr>
+            </thead>
+            <tbody>
+          <?php endif; ?>
+    
+<!-- If User chooses Both -->
+<?php if(isset($username) && isset($_POST['repos']) && isset($_POST['followers'])){ ?>
```

Comment:
არ ვიცი რამდენად გინდოდა, მაგრამ აქაც შეგიძლია ისეთივე სინტაქსი გამოიყენო როგორიც foreach-ზე გაქვს, ანუ `<?php if (condition) : ?>`  (for loop-ის შემთხვევაშიც მოსულა) .

Reply:
კი მასე ჯობს ჩავასწორებ